CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ UNRELEASED
 ----------
 
 * [ [#1827](https://github.com/digitalfabrik/integreat-cms/issues/1827) ] Make Multi-language-XLIFF export popup scrollable
+* [ [#1807](https://github.com/digitalfabrik/integreat-cms/issues/1807) ] Remove Short description field from POI form
```

Comment:
```suggestion
* [ [#1807](https://github.com/digitalfabrik/integreat-cms/issues/1807) ] Remove short description field from POI form
```
Found a small typo, but I admit that this is unnecessarily petty :laughing:

Reply:
Fixed, thanks! 👌